rkt/enter.go

```diff
@@ -0,0 +1,135 @@
+//+build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/appc/spec/schema"
+	"github.com/appc/spec/schema/types"
+	rktpath "github.com/coreos/rocket/path"
+	"github.com/coreos/rocket/pkg/lock"
+	"github.com/coreos/rocket/stage0"
+)
+
+var (
+	cmdEnter = &Command{
+		Name:    "enter",
+		Summary: "Enter the namespaces of a rkt job",
```

Comment:
What is a "job"? How about "enter the namespaces of an app within a rocket container"?